core/src/com/unciv/ui/cityscreen/CitizenManagementTable.kt

```diff
@@ -0,0 +1,64 @@
+package com.unciv.ui.cityscreen
+
+import com.badlogic.gdx.graphics.Color
+import com.badlogic.gdx.scenes.scene2d.Touchable
+import com.badlogic.gdx.scenes.scene2d.ui.Table
+import com.unciv.logic.city.CityFocus
+import com.unciv.models.stats.Stat
+import com.unciv.models.translations.tr
+import com.unciv.ui.images.ImageGetter
+import com.unciv.ui.utils.*
+
+class CitizenManagementTable(val cityScreen: CityScreen): Table() {
+    private val innerTable = Table()
+    val city = cityScreen.city
+
+    init {
+        innerTable.background = ImageGetter.getBackground(ImageGetter.getBlue().darken(0.5f))
+        add(innerTable).pad(2f).fill()
+        background = ImageGetter.getBackground(Color.WHITE)
+    }
+
+    fun update(visible: Boolean = false) {
+        innerTable.clear()
+
+        if (!visible) {
+            isVisible = false
+            return
+        }
+        isVisible = true
+
+        val focusLabel = "Default Focus".toLabel().addBorder(5f, Color.CLEAR)
+        val defaultCell = Table()
+        defaultCell.add(focusLabel)
```

Comment:
Instead of adding a 'clear border' you can defaultCell.add(focusLabel).pad(5f) - saves unnecessary objects